.travis.yml

```diff
@@ -17,9 +17,9 @@ before_install:
 

```

Comment:
You've got bad line endings too... my PR #8 added a `.gitattributes` file, maybe that should merge first?